pex/interpreter.py

```diff
@@ -305,6 +320,19 @@ class IdentificationError(Error):
     class InterpreterNotFound(Error):
         pass
 
+    @staticmethod
+    def latest_release_of_min_compatible_version(interps):
+        # type: (Sequence[PythonInterpreter]) -> PythonInterpreter
+        """Find the minimum major version, but use the most recent patch version within that minor
+        version.
+
+        That is, 3.6.1 < 3.6.0 < 3.7.*.
```

Comment:
I wouldn't use `<` in this example, since again this is not intended to be thought of as an ordering, per se.

Reply:
So just say something like "3.6.1 is selected over 3.6.0, but 3.6.0 is selected over 3.7.*".